File: C++/nilai_akhir.cpp
#include <iostream>
using namespace std;

int main (){
    float tugas, uts, uas, nilai_akhir;

    cout << "masukan nilai tugas: ";
    cin >> tugas;
    cout << "masukan nilai uts: ";
    cin >> uts;
    cout << "masukan nilai uas: ";
    cin >> uas;

    nilai_akhir = (0.3 * tugas) + (0.3 * uts) + (0.4 * uas);
    cout << "nilai akhir: " << nilai_akhir << endl;

    if (nilai_akhir >=60)
    {
        cout << "Status : Lulus" << endl;
    }
    else
    {
        cout << "Status : Tidak Lulus" << endl;
    }

    return 0;
}

File: Java/nilai_akhir.java
import java.util.Scanner;
public class nilai_akhir {
    public static void main(String[] args) {
    Scanner sc = new Scanner(System.in);
    System.out.print("masukan nilai tugas : ");
    double tugas = sc.nextDouble();
    System.out.print("masukan nilai uts : ");
    double uts = sc.nextDouble();
    System.out.print("masukan nilai uas : ");
    double uas = sc.nextDouble();

    double nilai_akhir = (0.3 * tugas) + (0.3 * uts) + (0.4 * uas);

    System.out.println("nilai akhir: " + nilai_akhir);
    if (nilai_akhir >= 60) {
        System.out.println("Status : Lulus");
    } else {
        System.out.println("Status : Tidak Lulus");
    }

    sc.close();
}
}